src/scripts/main.js

```diff
@@ -1 +1,14 @@
 'use strict';
+
+const images = document.querySelector('#thumbs');
+const largeImg = document.querySelector('#largeImg');
+
+images.onclick = function () {
```

Comment:
It seems like you're using the global 'event' object in your onclick function. While this might work in some browsers, it's not a standard feature and might not work everywhere. I'd recommend explicitly defining the event as an argument to your function, like this: `images.onclick = function (event) {...}`. This way, you're guaranteed to have access to the event object inside your function.

Reply:
idk what to do